### PR TITLE
feat: add dashboard lightweight card

### PR DESCRIPTION
**Pull request by kertuilves**
https://app.clickup.com/t/861n0qm47

### CODE REVIEW

**Comment by heshfekry** [comment]
@kertuilves just confirming that this is the component 

https://deploy-preview-289--conversionxl-aybolit.netlify.app/?path=/story/cxl-ui-cxl-card--cxl-course

**Comment by heshfekry** [comment]
Hey @kertuilves 

Would be good to see multiple cards in the same view so we can assess how they work together? [Similar to this.](https://deploy-preview-289--conversionxl-aybolit.netlify.app/?path=/story/cxl-ui-cxl-vaadin-accordion--cxl-vaadin-accordion-theme-archive)


Right now it seems to take full width,

image

 so unclear how it would function with multiple cards
 
 
image


Additionally
1.  would be good to see the indicator.
2.  A problem came up yday when we where working on roadmaps from teams and personal. [We probably need to have tags on there too indicating if the course has been selected in roadmap or not.](https://www.figma.com/file/RTj2PtUSrCOUFtljOiq0W3/Dashboard?node-id=445%3A39255&mode=dev)
- I would change the label from Team > Team Roadmap and from Personal > Personal Roadmap

image

**Review by pawelkmpt — CHANGES_REQUESTED** [review]
Icon is cut

Screenshot 2023-07-05 at 12 03 37

**Comment by pawelkmpt** [comment]
> A problem came up yday when we where working on roadmaps from teams and personal. [We probably need to have tags on there too indicating if the course has been selected in roadmap or not.](https://www.figma.com/file/RTj2PtUSrCOUFtljOiq0W3/Dashboard?node-id=445%3A39255&mode=dev)

Use vaadin badge for tags.

**Comment by kertuilves** [comment]
Hey @heshfekry.
1. I modified the Course Card canvas layout and now the icon is [visible](https://deploy-preview-289--conversionxl-aybolit.netlify.app/?path=/story/cxl-ui-cxl-card--cxl-course&args=State:new)
2. Course Card now has another `with-roadmaps` [state](https://deploy-preview-289--conversionxl-aybolit.netlify.app/?path=/story/cxl-ui-cxl-card--cxl-course&args=State:new)
3. I added an example under [cxl-tabs-slider](https://deploy-preview-289--conversionxl-aybolit.netlify.app/?path=/story/cxl-ui-cxl-tabs-slider--cxl-course-slider) to see multiple Course Cards in the same view

**Comment by heshfekry** [comment]
> Hey @heshfekry.
> 
> 1. I modified the Course Card canvas layout and now the icon is [visible](https://deploy-preview-289--conversionxl-aybolit.netlify.app/?path=/story/cxl-ui-cxl-card--cxl-course&args=State:new)
> 2. Course Card now has another `with-roadmaps` [state](https://deploy-preview-289--conversionxl-aybolit.netlify.app/?path=/story/cxl-ui-cxl-card--cxl-course&args=State:new)
> 3. I added an example under [cxl-tabs-slider](https://deploy-preview-289--conversionxl-aybolit.netlify.app/?path=/story/cxl-ui-cxl-tabs-slider--cxl-course-slider) to see multiple Course Cards in the same view

Looking good. The size seems to change with different states, unclear how that plays with multiple cards in the same view.

**Comment by kertuilves** [comment]
@heshfekry 

> Looking good. The size seems to change with different states, unclear how that plays with multiple cards in the same view.

I kept the card itself really basic and didn't constrain its component width. In my opinion this should be done when adding the card to the specific layout. In our example when adding it to the slider in the accordion. And those slider styles and card widths are right now visible in the [accordion](https://deploy-preview-288--conversionxl-aybolit.netlify.app/?path=/story/cxl-ui-cxl-vaadin-accordion--cxl-vaadin-accordion-theme-category) PR.

**Comment by heshfekry** [comment]
> @heshfekry
> 
> > Looking good. The size seems to change with different states, unclear how that plays with multiple cards in the same view.
> 
> I kept the card itself really basic and didn't constrain its component width. In my opinion this should be done when adding the card to the specific layout. In our example when adding it to the slider in the accordion. And those slider styles and card widths are right now visible in the [accordion](https://deploy-preview-288--conversionxl-aybolit.netlify.app/?path=/story/cxl-ui-cxl-vaadin-accordion--cxl-vaadin-accordion-theme-category) PR.

This makes sense. Does this apply to height too? for example when you introduce the tags or the names etc take more space.

**Comment by kertuilves** [comment]
> > @heshfekry
> > > Looking good. The size seems to change with different states, unclear how that plays with multiple cards in the same view.
> > 
> > 
> > I kept the card itself really basic and didn't constrain its component width. In my opinion this should be done when adding the card to the specific layout. In our example when adding it to the slider in the accordion. And those slider styles and card widths are right now visible in the [accordion](https://deploy-preview-288--conversionxl-aybolit.netlify.app/?path=/story/cxl-ui-cxl-vaadin-accordion--cxl-vaadin-accordion-theme-category) PR.
> 
> This makes sense. Does this apply to height too? for example when you introduce the tags or the names etc take more space.

Yes. So all the cards heights will be the same and it depends on the highest card. I kept it the same way as it is for the slider cards by default. 

Screenshot 2023-07-05 at 15 19 00

**Comment by heshfekry** [comment]
@freudFlintstone pulling you in here because you are working on upgrading the full card.

You mentioned that this could all be the same component with an attribute. Does it make sense you take over all the card components?

**Comment by heshfekry** [comment]
@freudFlintstone lets continue as is with now as separate components ill make another CU card to revisit the combination project at the end once we have everything in place.

File: packages/cxl-ui/scss/global/cxl-card.scss [hunk 1]
```diff
@@ -24,4 +24,58 @@ cxl-card {
...
+      align-items: flex-start;
+      justify-content: space-between;
+
+      > vaadin-icon {
+        position: absolute;
+        top: calc(var(--lumo-space-m) * -1);
+        right: 0;
+        width: calc(var(--lumo-space-m) * 2);
+        height: calc(var(--lumo-space-m) * 2);
+        padding: 6px;
```
[the remark this answers is not in the document]
Comment (pawelkmpt) [reply packages/cxl-ui/scss/global/cxl-card.scss]: I meant just `padding` ;) but as long as we don't have more icons it's fine

File: packages/storybook/cxl-ui/cxl-card/[theme=cxl-course].stories.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+CXLCourse.args = {
+  el: { cxl_hybrid_attr_post: { '@attributes': {} } },
+  State: 'default',
+};
+
+CXLCourse.argTypes = {
+  State: {
+    control: {
+      options: ['default', 'new', 'completed', 'with-roadmaps'],
```
Comment (pawelkmpt) [inline packages/storybook/cxl-ui/cxl-card/[theme=cxl-course].stories.js:59]: I suggest to change `with-roadmaps` to `with-badges` to make it more universal as badges might be used also for tags etc.

And question: is it possible to apply multiple states at the same time?

**Review by pawelkmpt — CHANGES_REQUESTED** [review]

File: packages/storybook/cxl-ui/cxl-card/[theme=cxl-course].stories.js (hunk 2 above)
Comment (kertuilves) [reply packages/storybook/cxl-ui/cxl-card/[theme=cxl-course].stories.js]: Changed the roadmaps -> badges and now it's possible to select multiple [states](https://deploy-preview-289--conversionxl-aybolit.netlify.app/?path=/docs/cxl-ui-cxl-card--cxl-course)

File: packages/storybook/cxl-ui/cxl-card/[theme=cxl-course].stories.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+          ${stateArray.map(state =>
+            state === 'with-badges'
+            ? html`
+              <div class="entry-summary">
+                <span theme="badge primary pill">Team Roadmap</span>
+                <span theme="badge secondary pill">Personal Roadmap</span>
+              </div>`
+            : html``
+          )}
+        </div>
```
Comment (pawelkmpt) [inline packages/storybook/cxl-ui/cxl-card/[theme=cxl-course].stories.js:49]: I'm concerned about the logic being present in storybook instead of encapsulating it into the component.

As I see, adding state param eg. `cxl-course-new` will not make new icon visible. It'll just apply specific styles, but if icon is not present... card will look the same as without this state.

For completed the same: we need to manually put an icon instead of just adding `cxl-course-completed` theme element.

Which makes me think: why do we even need to define states?

@lkraav what's your opinion?

**Review by pawelkmpt — COMMENTED** [review]
Looking good. Concerns about logic.

File: packages/storybook/cxl-ui/cxl-card/[theme=cxl-course].stories.js (hunk 3 above)
Comment (pawelkmpt) [reply packages/storybook/cxl-ui/cxl-card/[theme=cxl-course].stories.js:49]: With badges it should just work out of the box: put badges and they're displayed nicely without a need to add theme name.

Comment (lkraav) [reply packages/storybook/cxl-ui/cxl-card/[theme=cxl-course].stories.js:49]: > As I see, adding state param eg. `cxl-course-new` will not make new icon visible. It'll just apply specific styles, but if icon is not present... card will look the same as without this state.
> 
> For completed the same: we need to manually put an icon instead of just adding `cxl-course-completed` theme element.
> 
> Which makes me think: why do we even need to define states?

I think I agree - if possible, we should aim for an atomic (single) operation to achieve our goal. "Need to know multiple steps" is road to confusion.

Comment (pawelkmpt) [reply packages/storybook/cxl-ui/cxl-card/[theme=cxl-course].stories.js:49]: Alright, @kertuilves please encapsulate these states in the component.

`new` -> icon appears (without a need to put icon manually)

`completed` -> checkmark (without a need to put icon manually)

`with-badges` -> drop state; badges need to look good without need to add additional theme, just `cxl-course`

File: packages/cxl-lumo-styles/package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@conversionxl/cxl-lumo-styles",
-  "version": "1.6.3",
+  "version": "1.6.4",
```
Comment (anoblet) [inline packages/cxl-lumo-styles/package.json:3]: I'm not sure it's best practice to bump package versions with each PR. If so, it would be better in a separate commit.

@pawelkmpt @lkraav Thoughts?

**Review by anoblet — CHANGES_REQUESTED** [review]

**Comment by pawelkmpt** [comment]
@kertuilves it seems like you squashed your changes with existing commit during rebase. Please correct it. Are you familiar with `git reflog`?

**Comment by pawelkmpt** [comment]
@kertuilves you pushed wrong code here `cxl-stats`

**Comment by kertuilves** [comment]
> @kertuilves you pushed wrong code here `cxl-stats`

@pawelkmpt The `cxl-stats` has been merged to feat/dashboard and when I rebased this branch from feat/dashboard and solved the merge conflicts it was supposed to come. No?

**Comment by pawelkmpt** [comment]
> @pawelkmpt The `cxl-stats` has been merged to feat/dashboard and when I rebased this branch from feat/dashboard and solved the merge conflicts it was supposed to come. No?

Everything is fine. My bad. I looked at _Compare_ option https://github.com/conversionxl/aybolit/compare/9f2bfc79dfe1adc13fb54768514c200119990f29..c9487b473530c54471bd36a12314245061c9e8c6, shown stats component there and I had an impression it's pushed here.

Sorry for that

**Comment by pawelkmpt** [comment]
@kertuilves taking into account Slack threads about the way of developing components:
* https://cxlworld.slack.com/archives/C01HXUNGEPM/p1689060114569099
* https://cxlworld.slack.com/archives/C01HXUNGEPM/p1689084292805349

I think we need change in this code. @freudFlintstone has almost completed `<cxl-course-card>` and you can look at his code here: https://github.com/conversionxl/aybolit/commit/3e46d1286e2d7c4a8b457581a4b58b42be0f679d#diff-5e748d1f0301dffd4bb4521b9dbe089c8a76e34023e06745e8848cd542447a6e

Main difference: component has set of attributes and properties which are encapsulated and rendered in specific way.

Good thing is that you can re-use styles you already created :)

What's next:
* keep `cxl-card` as is,
* created new component `cxl-light-card`.

**Comment by kertuilves** [comment]
@pawelkmpt I created a separate component for light-card. How should I pass the badges data? Do we know how will the different roadmaps themes be defined? (will work on it after vacation)

File: packages/cxl-ui/src/components/cxl-light-card.js [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+    return [cxlLightCardStyles];
+  }
+
+  @property({ type: String }) id = '';
+
+  @property({ type: String }) theme = 'light-card';
+
+  @property({ type: String }) title = 'Account based marketing';
+
+  @property({ type: String }) time = '3h 00min';
```
Comment (pawelkmpt) [inline packages/cxl-ui/src/components/cxl-light-card.js:19]: Default should be empty string

File: packages/cxl-ui/src/components/cxl-light-card.js [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+
+  @property({ type: String }) instructorPrefix = 'By:';
+
+  @property({ type: String }) instructor = 'Ton Wesseling';
```
Comment (pawelkmpt) [inline packages/cxl-ui/src/components/cxl-light-card.js:23]: Default should be empty string

File: packages/cxl-ui/src/components/cxl-light-card.js [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+
+  @property({ type: String }) avatar =
```
Comment (pawelkmpt) [inline packages/cxl-ui/src/components/cxl-light-card.js:25]: Default should be empty string

File: packages/cxl-ui/src/components/cxl-light-card.js [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+          <div>
+            <div class="entry-title">
+              ${this.title}
+              <vaadin-icon icon="lumo:checkmark"></vaadin-icon>
+            </div>
+            <div class="entry-byline">
+              <div><vaadin-icon icon="lumo:clock"></vaadin-icon>${this.time}</div>
+              <div>${this.instructorPrefix}&nbsp;${this.instructor}</div>
+            </div>
+            <slot name="badges"></slot>
```
Comment (pawelkmpt) [inline packages/cxl-ui/src/components/cxl-light-card.js:44]: > How should I pass the badges data? Do we know how will the different roadmaps themes be defined?

I think we should rename it to `footer` and let us pass any HTML we want. It will be more flexible and easier to manage. Today requirement is for badges but it might be also text tags etc.

File: packages/cxl-ui/src/components/cxl-light-card.js [hunk 9]
```diff
@@ -0,0 +1,51 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { LitElement, html } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import '@conversionxl/cxl-lumo-styles';
+import cxlLightCardStyles from '../styles/cxl-light-card-css.js';
+
+@customElement('cxl-light-card')
+export class CXLCourseCardElement extends LitElement {
```
Comment (pawelkmpt) [inline packages/cxl-ui/src/components/cxl-light-card.js:8]: I think this should have a different name

**Review by pawelkmpt — CHANGES_REQUESTED** [review]

File: packages/cxl-ui/src/components/cxl-light-card.js [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+
+  @property({ type: String }) time = '';
+
+  @property({ type: String }) instructorPrefix = 'By:';
+
+  @property({ type: String }) instructor = '';
+
+  @property({ type: String }) avatar = '';
+
+  @property({ type: Boolean }) hasFooter = false;
```
Comment (pawelkmpt) [inline packages/cxl-ui/src/components/cxl-light-card.js:27]: Is it needed to display footer? Can't we just put content into slot and render it automatically and it's there's nothing to display, card lives without footer?

File: packages/cxl-ui/src/components/cxl-light-card.js [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+
+  render() {
+    return html`
+      <div class="container">
+        <header class="entry-header">
+          <vaadin-icon icon="cxl:new"></vaadin-icon>
```
Comment (pawelkmpt) [inline packages/cxl-ui/src/components/cxl-light-card.js:33]: "New" is used as a property in #294 and #293 and we should make it same here

File: packages/cxl-lumo-styles/scss/badge.scss [hunk 12]
```diff
@@ -7,6 +7,11 @@
   color: var(--lumo-primary-text-color);
 }
 
+[theme~='badge'][theme~='secondary'] {
+  color: var(--lumo-primary-contrast-color);
+  background-color: var(--cxl-color-brand-blue);
+}
+
```
Comment (pawelkmpt) [inline packages/cxl-lumo-styles/scss/badge.scss:14]: Please make it separate commit as it's badge feature

**Review by pawelkmpt — CHANGES_REQUESTED** [review]
Navigation differs from Figma design

it is:
Screenshot 2023-07-27 at 15 57 23

should be:
Screenshot 2023-07-27 at 15 57 08

File: packages/cxl-ui/src/components/cxl-light-card.js (hunk 11 above)
Comment (kertuilves) [reply packages/cxl-ui/src/components/cxl-light-card.js:33]: I made the "new" as a property and it made sense to use the same pattern with completed card. So now the "completed" is also a separate property and I removed the themes because these are not necessary anymore.

**Comment by kertuilves** [comment]
> Navigation differs from Figma design

I made the slider changes now here, too. But this was already discussed above that those slider styles were already under #288. And the slider here was just to show how multiple cards together would behave.

File: packages/cxl-lumo-styles/scss/badge.scss (hunk 12 above)
Comment (pawelkmpt) [reply packages/cxl-lumo-styles/scss/badge.scss:14]: @kertuilves following it up

File: packages/cxl-lumo-styles/src/icons.js [hunk 13]
```diff
@@ -53,6 +53,9 @@ $documentContainer.innerHTML = `
           <path d="M5.614.416a4.538 4.538 0 011.824.086A4.259 4.259 0 005.614.416zM2.322 12.449l.002.009c-.747-1.19-.94-.39-.97-.224-.587-1.581-.96-2.743-.96-2.743a10.93 10.93 0 01-.15-.61L.22 8.77c-.034-.168-.063-.33-.09-.484v-.003a16.671 16.671 0 01-.077-.583C-.04 6.725.04 5.89.23 5.176c.011-.039.02-.08.034-.117.01-.034.02-.07.033-.102.335-1.039.72-1.273.72-1.273 1-.892.963.434.963.434-1.05 2.304.227 7.84.338 8.312.003.006.003.013.005.019zM3.472 1.432A4.452 4.452 0 015.24.492a4.102 4.102 0 00-1.768.94z"/>
           <path d="M1.355 12.234l-.006.029A17.108 17.108 0 01.243 8.88c.043.2.094.403.15.61 0 0 .374 1.16.962 2.743zM.054 7.7C-.064 6.576.05 5.967.05 5.967c.055-.3.116-.563.18-.79-.19.714-.269 1.549-.175 2.525z"/>
         </g>
+        <g id="new" width="20" height="20" viewBox="0 0 20 20">
```
Comment (pawelkmpt) [inline packages/cxl-lumo-styles/src/icons.js:56]: I think this branch needs rebase on top of `feat/dashboard`. the other branch already have "new" icon https://github.com/conversionxl/aybolit/blob/e4a46aa4647d7882365e5a79aaf7b08ce7481d93/packages/cxl-lumo-styles/src/icons.js#L56-L58

**Review by pawelkmpt — CHANGES_REQUESTED** [review]
Please put badge code into one commit and rest into another

File: packages/cxl-lumo-styles/scss/global.scss [hunk 14]
```diff
@@ -8,6 +8,7 @@ html {
   --cxl-wrap-padding: var(--lumo-space-m);
   --cxl-color-light-gray: hsla(0, 0%, 96%, 1);
   --cxl-color-black-50pct: hsla(0, 0%, 0%, 0.5);
+  --cxl-color-brand-blue: hsla(214, 61%, 25%, 1);
```
Comment (pawelkmpt) [inline packages/cxl-lumo-styles/scss/global.scss:11]: This color should be in the other branch

**Review by pawelkmpt — CHANGES_REQUESTED** [review]

**Review by pawelkmpt — APPROVED** [review]
Good job 👍 
Next step: refactor to use base class

File: packages/cxl-ui/src/components/cxl-base-card.js [hunk 15]
```diff
@@ -50,16 +54,19 @@ export class CXLBaseCardElement extends LitElement {
...
   }
 
   _renderHeaderAttributes() {
     return html`
       <div class="attributes">
-        <cxl-time
-          time=${this.time}
-          ?show-icon=${!!(this.theme.toLowerCase() === 'course')}
-        ></cxl-time>
+        <cxl-time time=${this.time} ?show-icon=${this.showTimeIcon}></cxl-time>
```
Comment (pawelkmpt) [inline packages/cxl-ui/src/components/cxl-base-card.js:69]: Changing this logic impacts `<cxl-course-card>`. You set it ` @state() showTimeIcon = true;` in the component but it's different then it was originally. Please revisit original code and adjust

File: packages/cxl-ui/scss/cxl-base-card.scss [hunk 16]
```diff
@@ -157,8 +157,8 @@ header {
 :host([new]) {
   .badge-new {
     position: absolute;
-    top: calc(-1 * var(--lumo-space-s));
-    right: calc(-1 * var(--lumo-space-s));
+    top: calc(-1 * var(--lumo-space-m));
+    right: 0;
```
Comment (pawelkmpt) [inline packages/cxl-ui/scss/cxl-base-card.scss:161]: I see there's difference in Figma design between different cards. I experimented with values and I think original style here makes more sense, so please revert.

File: packages/cxl-ui/scss/cxl-time.scss [hunk 17]
```diff
@@ -1,5 +1,6 @@
 :host {
   display: flex;
+  align-items: center;
```
Comment (pawelkmpt) [inline packages/cxl-ui/scss/cxl-time.scss:3]: 👍

**Review by pawelkmpt — CHANGES_REQUESTED** [review]

File: packages/cxl-ui/scss/cxl-base-card.scss [hunk 18]
```diff
@@ -157,8 +157,8 @@ header {
 :host([new]) {
   .badge-new {
     position: absolute;
-    top: calc(-1 * var(--lumo-space-m));
-    right: 0;
+    top: calc(-1 * var(--lumo-space-s));
+    right: calc(-1 * var(--lumo-space-s));
```
Comment (pawelkmpt) [inline packages/cxl-ui/scss/cxl-base-card.scss]: This should go into 404b91d37d4eb09fc60ca27f4c3df7a2c2c8da3c

File: packages/cxl-ui/src/components/cxl-base-card.js [hunk 19]
```diff
@@ -33,7 +33,7 @@ export class CXLBaseCardElement extends LitElement {
 
   @property({ type: Boolean, reflect: true }) completed = false;
 
-  @property({ type: Boolean, reflect: true }) showTimeIcon = false;
+  @property({ type: Boolean, reflect: true, attribute: 'show-time-icon' }) showTimeIcon = false;
```
Comment (pawelkmpt) [inline packages/cxl-ui/src/components/cxl-base-card.js:36]: This should go into 404b91d37d4eb09fc60ca27f4c3df7a2c2c8da3c

File: packages/cxl-ui/src/components/cxl-course-card.js [hunk 20]
```diff
@@ -14,8 +14,6 @@ export class CXLCourseCardElement extends CXLBaseCardElement {
 
   @state() _moreHasChildren = false;
 
-  @state() showTimeIcon = true;
-
```
Comment (pawelkmpt) [inline packages/cxl-ui/src/components/cxl-course-card.js]: This should go into 404b91d37d4eb09fc60ca27f4c3df7a2c2c8da3c

File: packages/storybook/cxl-ui/cxl-course-card/[theme=playbook].stories.js [hunk 21]
```diff
@@ -1,23 +1,24 @@
...
   theme: 'playbook',
   name: 'The Why’s and How’s of Marketing Attribution',
   time: '12 days ago',
-  description: 'Master the strategies, tactics, metrics, and wisdom you need to become an ABM leader and accelerate the growth of your company and of your career.',
+  description:
+    'Master the strategies, tactics, metrics, and wisdom you need to become an ABM leader and accelerate the growth of your company and of your career.',
   contentTags: [],
-  tags: ['Growth Marketing']
+  tags: ['Growth Marketing'],
+  showTimeIcon: false,
```
Comment (pawelkmpt) [inline packages/storybook/cxl-ui/cxl-course-card/[theme=playbook].stories.js:23]: This should go into 404b91d37d4eb09fc60ca27f4c3df7a2c2c8da3c

File: packages/storybook/cxl-ui/cxl-course-card/[theme=training].stories.js [hunk 22]
```diff
@@ -1,23 +1,24 @@
...
   theme: 'training',
   name: 'A/B testing mastery',
-  description: 'Understand testing approaches that work (and pitfalls that don’t) to get more wins and insights from optimization efforts.',
+  description:
+    'Understand testing approaches that work (and pitfalls that don’t) to get more wins and insights from optimization efforts.',
   tags: ['CRO', 'Branding'],
   contentTags: [],
-  more: ''
+  more: '',
+  showTimeIcon: false,
```
Comment (pawelkmpt) [inline packages/storybook/cxl-ui/cxl-course-card/[theme=training].stories.js:23]: This should go into 404b91d37d4eb09fc60ca27f4c3df7a2c2c8da3c

File: packages/storybook/cxl-ui/cxl-course-card/[theme=video].stories.js [hunk 23]
```diff
@@ -1,24 +1,25 @@
...
-  description: 'Understand testing approaches that work (and pitfalls that don’t) to get more wins and insights from optimization efforts.',
+  description:
+    'Understand testing approaches that work (and pitfalls that don’t) to get more wins and insights from optimization efforts.',
   tags: ['CRO'],
-  contentTags: ['running experiments', 'customer base studies', 'prioritization' ],
+  contentTags: ['running experiments', 'customer base studies', 'prioritization'],
   time: '4 days ago',
-  new: true
+  new: true,
+  showTimeIcon: false,
```
Comment (pawelkmpt) [inline packages/storybook/cxl-ui/cxl-course-card/[theme=video].stories.js:24]: This should go into 404b91d37d4eb09fc60ca27f4c3df7a2c2c8da3c

File: packages/storybook/cxl-ui/cxl-course-card/template.js [hunk 24]
```diff
@@ -1,38 +1,51 @@
 import { html } from 'lit';
 
-const renderTags = (tags, slot) => tags.map((tag, i) => html`${i > 0 ? html`<span slot=${slot}> | </span>` : ''}<span class="tag" slot=${slot} title="${tag}">${tag}</span>`);
+const renderTags = (tags, slot) =>
```
Comment (pawelkmpt) [inline packages/storybook/cxl-ui/cxl-course-card/template.js:3]: This should go into 404b91d37d4eb09fc60ca27f4c3df7a2c2c8da3c

**Review by pawelkmpt — CHANGES_REQUESTED** [review]
@kertuilves I marked couple of changes which should go into "improve base card" commit. For storybook templates it's not just a line I left the comment but all changes which are caused by `showTimeIcon` property introduction.

Light card itself looks good

**Review by pawelkmpt — APPROVED** [review]
